RELEASE-NOTES.md

```diff
@@ -55,6 +55,8 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
   candidate BVH bins that intersect each bounding box.
 - Added an `axom-config.cmake` file to axom's installation to streamline incorporating axom
   into user applications. See `<axom-install>/examples/axom` for example usages.
+- Added a new built-in TPL, Sol, for a fast, simple C++ and Lua Binding 
+  (automatically enabled when LUA_DIR is found)
```

Comment:
Please add: 
* Which version of SOL we are using
* that SOL requires `C++14`
* a link to SOL.